### DISCUSSION TITLE
Ɲɲ

### DISCUSSION THREAD

**Discussion by Christine-Gertsch** [koeberlin/Designing-Latin-S#5] [Q&A] [answered]
Is Ɲɲ meant to have an emphasized hook or can it be short for tighter spacing?

<img width="1138" alt="Pangea-PanAfricanGlyphs" src="https://github.com/koeberlin/Designing-Latin-S/assets/146826184/4d4390df-7c0e-4cf3-9e7d-7676fa52fe1f">

**Comment by koeberlin** [comment, marked as answer]
@Christine-Gertsch Oops! I think hooks should be the same here, sorry!

**Reply by Christine-Gertsch** [reply]
thanks!
